app/src/main/res/values/strings.xml

```diff
@@ -731,7 +731,7 @@
          speech-to-text recognition. -->
     <string name="voice_search_tooltip">Voice Search</string>
 
-    <!-- This string is for the tooltip that appears upon hovering the 'User Agent' button in the
+    <!-- This string is for the tooltip that appears upon hovering the 'Request desktop site' button in the
```

Comment:
The initial comment and string that landed was "Switch User Agent". Since this changes, there should be a new String ID to reflect that.

Reply:
@Delphine Should be ok no

Reply:
Yes, thanks!